docs/Compile.rst

```diff
@@ -200,39 +280,141 @@ On Windows, DFHack replaces the SDL library distributed with DF.
 
 Dependencies
 ------------
-You will need some sort of Windows port of git, or a GUI. Some examples:
+You will need the following:
+
+* Microsoft Visual Studio 2010 SP1, with the C++ language
+* Git
+* CMake
+* Perl with XML::LibXML and XML::LibXSLT
+
+  * It is recommended to install StrawberryPerl, which includes both.
+
+Microsoft Visual Studio 2010 SP1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The free Express version is sufficient.
+
+You can grab it from `Microsoft's site <http://download.microsoft.com/download/1/E/5/1E5F1C0A-0D5B-426A-A603-1798B951DDAE/VS2010Express1.iso>`_.
+
+You'll also need the Visual Studio 2010 SP1 update, which is obtained from
+Windows Update. After installing Visual Studio, be sure to go to Windows Update
+and check for and install the SP1 update. If no update is found, check that
+your Windows Update settings include "Updates from all Microsoft products".
+
+You can confirm whether you have SP1 by opening the Visual Studio 2010 IDE
+and selecting About from the Help menu.  If you have SP1 it will have *SP1Rel*
+at the end of the version number, for example: *Version 10.0.40219.1 SP1Rel*
+
+It is vital that you use SP1 as, while building with the original release
+of Visual Studio 2010 (RTM) may succeed, it will result in non-working DFHack
+binaries that crash when connecting to Dwarf Fortress.
+
+Additional dependencies: installing with the Chocolatey Package Manager
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The remainder of dependencies - Git, CMake and StrawberryPerl - can be most
+easily installed using the Chocolatey Package Manger. Chocolatey is a
+\*nix-style package manager for Windows. Think "``apt-get`` for Windows."
+
+Chocolatey is the recommended way of installing the required dependencies
```

Comment:
OK updated it in the next commit, now it is "a preferred way of installing the required dependencies".  And I put in a brief note on the very small disk requirements (I checked on the Windows 7 install in which I installed Chocolatey + just the 3 DFHack dependencies, and disk usage is 7.5MB there.)  And similarly removed the strong recommendation for Chocolatey under the Manually Installing Dependencies section.